store/index.js

```diff
@@ -148,11 +159,71 @@ export const actions = {
         	"Assessment Tool"
 		];
 
+		// 0. This annotation information is default but we can swap out and reinitialize
+		// annotation data structures by calling 'initializeAnnotationDetails' with a new
+		// object containing annotation information for each category
+		const annotationDetails = [
+
+			{
+				id: 0,
+				category: "Age",
+				explanation: "This is an explanation for how to annotate age.",
+				options: {},
+				specializedComponent: "annot-age-values"
+			},
+			{
+				id: 1,
+				category: "Sex",
+                explanation: "This is an explanation for how to annotate sex.",
+				options: ["male", "female", "other", "missing value"],
+				specializedComponent: "annot-discrete-choices"
+			},
+			{
+				id: 2,
+				category: "Diagnosis",
+				explanation: "This is an explanation for how to annotate diagnosis.",
+				options: { mode: "row" },
+				specializedComponent: "annot-vocabulary"
+			},
+			{
+				id: 3,
+				category: "Assessment Tool",
+				explanation: "This is an explanation for how to annotate assessments.",
+				options: { mode: "column" },
+				specializedComponent: "annot-vocabulary"
+			}
+		];
+
 		// 1. Setup category-related data structures based on the given categories
-		commit("setupCategories", categories);	
+		commit("setupCategories", categories);
+
+		// 2. Setup annotation-related data structures based on the given categories\
+		commit("setupAnnotationDetails", annotationDetails);
 	},
 
 	// Tool navigation
+
+	enablePage(p_context, p_navData) {
```

Comment:
how about `initializePage`? Maybe not quite the right name, but this method seems to do more than just make the link clickable, particularly in the future?

Reply:
Perfect. 👌